sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/_input_outputs.py

```diff
@@ -12,6 +13,22 @@
 _INPUT_TYPE_FLOAT_CAP = "Float"
 
 
+class UIWidgetTypeEnum(Enum):
+    DEFAULT = "Default"
```

Comment:
How do we handle case that new UIWidgetType added?

Reply:
Because cannot call azure.ml.component to get the latest UIWidgetType, can all unsupported types be converted to str type?
But it will affact the validation result.